docs/_documentation/fundamentals/inversion-of-control-ioc.md

```diff
@@ -98,6 +98,17 @@ but this creates boilerplate code and in case you need another instance with a d
 Then at the moment of resolving the interface the implementation takes the same generic type parameter that the interface, e.g. if you resolve `var foo = Mvx.Resolve<IFoo<Bar1>>();` then `foo` will be of type `Foo<Bar1>`.
 As you can see this give us more flexibility and scalability because we can effortlessly change the generic type parameters at the moment of resolving the interface and we don't need to add anything to register the interface with a new generic type parameter.
 
+### Child Containers
+
+Sometimes you'd like to add some instances or types to an IoC Container for a specific purpose and not to the app-wide container. You can use Child Containers for that:
+
+    var container = Mvx.Resolve<IMvxIoCProvider>();
+    var childContainer = container.CreateChildContainer():
```

Comment:
Maybe we can add this method to `IMvxIoCProvider` instead of using an extension method?

Reply:
Thought about that for a moment. I initially added the extension method just for convenience as it simply returns an new container instance. But maybe one wants to customize the returned child container 🤔

Reply:
I think (I might be wrong) that having the method in the interface will make it easier to mock in unit tests.

Reply:
:+1: Removed the extension and added `CreateChildContainer` to the interface.